lib/Provider/MailService.php

```diff
@@ -0,0 +1,182 @@
+		if (isset($this->serviceAbilities[$value])) {
+			return (bool)$this->serviceAbilities[$value];
+		}
+		
+		return false;
+	}
+
+	/**
+	 * Retrieves a collection of what actions a service can perform
+	 *
+	 * @since 4.0.0
+	 *
+	 * @return array collection of abilities otherwise empty collection
+	 */
+	public function capabilities(): array {
+		return $this->serviceAbilities;
+	}
+
+	/**
+	 * Gets the localized human friendly name of this service
+	 *
+	 * @since 4.0.0
+	 *
+	 * @return string label/name of service (e.g. ACME Company Mail Service)
+	 */
+	public function getLabel(): string {
+		return $this->serviceLabel;
+	}
+
+	/**
+	 * Sets the localized human friendly name of this service
+	 *
+	 * @since 4.0.0
+	 *
+	 * @param string $value label/name of service (e.g. ACME Company Mail Service)
+	 *
+	 * @return self return this object for command chaining
+	 */
+	public function setLabel(string $value): self {
+		$this->serviceLabel = $value;
+		return $this;
+	}
+
+	/**
+	 * Gets the primary mailing address for this service
+	 *
+	 * @since 4.0.0
+	 *
+	 * @return IAddress mail address object
+	 */
+	public function getPrimaryAddress(): IAddress {
+		// retrieve and return primary service address
+		return $this->servicePrimaryAddress;
+	}
+
+	/**
+	 * Sets the primary mailing address for this service
+	 *
+	 * @since 4.0.0
+	 *
+	 * @param IAddress $value mail address object
+	 *
+	 * @return self return this object for command chaining
+	 */
+	public function setPrimaryAddress(IAddress $value): self {
+		$this->servicePrimaryAddress = $value;
+		return $this;
+	}
+
+	/**
+	 * Gets the secondary mailing addresses (aliases) collection for this service
+	 *
+	 * @since 4.0.0
+	 *
+	 * @return array<int, IAddress> collection of mail address object [IAddress, IAddress]
+	 */
+	public function getSecondaryAddresses(): array {
+		// retrieve and return secondary service addresses (aliases) collection
+		return $this->serviceSecondaryAddresses;
+	}
+
+	/**
+	 * Sets the secondary mailing addresses (aliases) for this service
+	 *
+	 * @since 4.0.0
+	 *
+	 * @param IAddress ...$value collection of one or more mail address object
+	 *
+	 * @return self return this object for command chaining
+	 */
+	public function setSecondaryAddresses(IAddress ...$value): self {
+		$this->serviceSecondaryAddresses = $value;
+		return $this;
+	}
+
+	/**
+	 * Construct a new fresh message object
+	 *
+	 * @since 30.0.0
+	 *
+	 * @return IMessage fresh message object
+	 */
+	public function initiateMessage(): IMessage {
+		return new Message();
+	}
+
+	/**
+	 * Sends an outbound message
+	 *
+	 * @since 4.0.0
+	 *
+	 * @param IMessage $message mail message object with all required parameters to send a message
+	 * @param array $options array of options reserved for future use
+	 *
+	 * @throws SendException on failure, check message for reason
+	 */
+	public function sendMessage(IMessage $message, array $option = []): void {
```

Comment:
```suggestion
	public function sendMessage(IMessage $message, array $options = []): void {
```

Or update the phpdoc.